RELEASE.md

```diff
@@ -7,15 +7,17 @@ Please follow the established format:
 -->
 # Next release
 
+# Release 9.1.0
+
 ## Major features and improvements
+- Include expandAllPipelines in initial state. (#1896)
+- Allow Kedro-Viz commands to run from any sub directory within Kedro project. (#1871)
 
 ## Bug fixes and other changes
 
 - Upgrade the gitpod workspace-full to a newer version which includes both Node 18 and Python 3.11.5. (#1862)
 - Refactor backend integration with Kedro by replacing bootstrap_project with configure_project. (#1796)
 - Enhance kedro-viz doc integration. (#1874)
```

Comment:
Can we also add this 
- Migrate from CircleCi to Github Actions. (#1876)
- Update settings panel design (#1875)